autoscheduler/frontend/src/package.json

```diff
@@ -49,6 +49,8 @@
     "prop-types": "^15.7.2",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
+    "react-google-button": "^0.5.3",
+    "react-google-login": "^5.0.7",
```

Comment:
Do you plan on using these in this PR? We shouldn't need the google log in until version 0.2 or 0.3